### PR TITLE
Add documentation framework

### LINKED ISSUE #37: Build the documentation framework
## Summary

Construct the basic documentation framework - not necessary fully filled out docs. 

## Proposed implementation

Use docusaurus

## Related to

#30

### PR DESCRIPTION
**Pull request by kcpevey**
Resolves #37

Changes proposed in this pull request:
- adds docusaurus
- adds a few documentation pages to start filling in the space (still needs work on content)

### CODE REVIEW

File: documentation/src/components/HomepageFeatures/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    Link,
+    Route,
+    Switch,
+  } from 'react-router-dom';
+
+
+const FeatureList = [
+  {
+    title: <Link to="/docs/getting-started/installation">Installation</Link>,
+    url: "/docs/getting-started/installation",
```
Comment (iameskild) [inline documentation/src/components/HomepageFeatures/index.js]:
```suggestion
    title: <Link to="/getting-started/installation">Installation</Link>,
```

File: documentation/src/components/HomepageFeatures/index.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    imageLink: "/docs/getting-started/installation",
+    description: (
+      <>
+        Getting started using the IB Manifest Util is straightforward and takes
+        less than 5 minutes to get set up.
+      </>
+    ),
+  },
+  {
+    title: <Link to="/docs/getting-started/installation">Update your IB Container</Link>,
```
Comment (iameskild) [inline documentation/src/components/HomepageFeatures/index.js]:
```suggestion
    title: <Link to="/getting-started/installation">Update your IB Container</Link>,
```

File: documentation/src/components/HomepageFeatures/index.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    Svg: require('@site/static/img/update.svg').default,
+    description: (
+      <>
+        Need to add a package? IB Manifest Util provides a single high-level command
+        for generating all the files you'll need to update your container repo!
+      </>
+    ),
+  },
+  {
+    title: <Link to="/docs/community/contributing">Community</Link>,
```
Comment (iameskild) [inline documentation/src/components/HomepageFeatures/index.js]:
```suggestion
    title: <Link to="/community/contributing">Community</Link>,
```

File: documentation/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
```
Comment (iameskild) [inline documentation/README.md:7]: Maybe add a note here saying that these commands need to be run in the `documentation` folder?

**Review by iameskild — CHANGES_REQUESTED** [review]
The links on the first page were broken for me but otherwise this looks great :)

File: documentation/src/components/HomepageFeatures/index.js (hunk 1 above)
Comment (iameskild) [reply documentation/src/components/HomepageFeatures/index.js]: I'm not sure we need the `url` key...

Comment (kcpevey) [reply documentation/src/components/HomepageFeatures/index.js]: good catch!

File: documentation/README.md (hunk 4 above)
Comment (kcpevey) [reply documentation/README.md:7]: yes, will do

**Review by iameskild — APPROVED** [review]
Looks great to me 🦖

**Pull request merged**